### PR TITLE
Add a test case for PATCH /staged with empty leg(s) in "transport_params"

### PR DESCRIPTION
**Pull request by garethsb-sony**
Empty objects in `"transport_params"` are valid per the schema, and necessary for example in order to be able to express patching of only parameters of the secondary leg of a two-legged Sender or Receiver, using a request like:

```json
{
  "transport_params": [
    {},
    {
      "rtp_enabled": false
    }
  ]
}
```

### CODE REVIEW

File: nmostesting/suites/IS0501Test.py [hunk 1]
```diff
@@ -477,6 +477,18 @@ def test_23(self, test):
         else:
             return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_23_01(self, test):
+        """Senders accept a patch request with empty leg(s) in transport parameters"""
+
+        if len(self.receivers) > 0:
```
Comment (jasperpeeters) [inline nmostesting/suites/IS0501Test.py]: `if len(self.senders) > 0` :)

Comment (garethsb-sony) [reply nmostesting/suites/IS0501Test.py]: Obviously I was just checking you were paying attention 🙄 
Thanks!!
